definitions/rnaseq_star_fusion.wdl

```diff
@@ -9,7 +9,7 @@ import "tools/index_cram.wdl" as ic
 import "tools/kallisto.wdl" as k
 import "tools/mark_duplicates_and_sort.wdl" as mdas
 import "tools/samtools_sort.wdl" as ss
-import "tools/star_align_fusion.wdl" as saf
+# import "tools/star_align_fusion.wdl" as saf
```

Comment:
remove commented line completely?

Reply:
Ugg, fixed it [here](https://github.com/sridhar0605/analysis-wdls/commit/b12601c0d110793c52fd171dc0a6e86f585bf775) thanks to catching this one.